src/routes/github/configuration/github-configuration-get.ts

```diff
@@ -153,7 +153,9 @@ export const GithubConfigurationGet = async (req: Request, res: Response, next:
 	req.log.debug("found github token");
 
 	if (!jiraHost) {
-		return next(new Error(Errors.MISSING_JIRA_HOST));
+		req.log.warn({ req, res }, Errors.MISSING_JIRA_HOST);
+		res.status(404).send(Errors.MISSING_JIRA_HOST);
+		return;
```

Comment:
Updated to `return next()`.
Just for my knowledge, whats the difference in `return` and `return next()`? Isn't the output the same?

Reply:
Yes, the output is the same but return next() makes sure that the execution stops after triggering the callback, just to avoid calling it again a second time.